app/assets/stylesheets/application.scss

```diff
@@ -4,6 +4,7 @@ $govuk-new-link-styles: true;
 // Components from govuk_publishing_components gem
 @import 'govuk_publishing_components/govuk_frontend_support';
 @import 'govuk_publishing_components/components/back-link';
+@import 'govuk_publishing_components/components/big-number';
```

Comment:
The new focus and focus + hover state for this seems unnecessarily busy with the extra underline. Did you consider mimicking the existing appearance for the component?

Before | After
------ | ------
![Screenshot 2021-09-24 at 07 54 35](https://user-images.githubusercontent.com/861310/134631399-58208669-caa7-4e6e-9ca2-a53aeabb804c.png) | ![Screenshot 2021-09-24 at 07 54 27](https://user-images.githubusercontent.com/861310/134631418-8cb98b6b-1513-444a-8d94-731c9376c387.png)

(this isn't the place for a longer conversation on this, so I'm going to approve regardless because I think using a component for this is a good idea)

Reply:
Good spot. I'll raise a PR on the gem soon as to refine this a bit more. I'll merge this in for now.